server/routes.go

```diff
@@ -733,6 +733,11 @@ func (s *Server) ListModelsHandler(c *gin.Context) {
 				return err
 			}
 
+			if strings.HasPrefix(filepath.Base(path), ".") {
```

Comment:
`filepath.Match` might be better

```suggestion
			if hidden, err := filepath.Match(".*", filepath.Base(rel); err != nil {
				return err
			} else if hidden {
				return nil
			}
```